### PR TITLE
GH130: Adjust GetTitle to only strip extension

### LINKED ISSUE #130: NormalizedPath GetTitle strips more than expected extentsion
https://github.com/statiqdev/Statiq.Framework/blob/ecaf1c7d183373416b8e597635801f15958d435a/src/core/Statiq.Common/IO/NormalizedPath.cs#L910

Currently GetTitle is expected to strip extension but strips more

```diff
In: 2020.09.29.1.htm
+ Expect: 2020.09.29.1
- Result: 2020
```
```diff
In: Statiq.Framework.sln.DotSettings
+ Expect: Statiq.Framework.Sln
- Result: Statiq
```

I'll send a PR with a fix, feel free to close if my expectations are wrong.

### PR DESCRIPTION
**Pull request by devlead**
* fixes #130

### CODE REVIEW

File: tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs [hunk 1]
```diff
@@ -1456,6 +1456,23 @@ public void OptimizesInstance()
                 // Then
                 result.FullPath.ShouldBe("a/b/c/abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz0123456789.html");
             }
+
+            [TestCase("2020.09.29.1.html", "2020.09.29.1")]
+            [TestCase("Statiq.Framework.sln.DotSettings", "Statiq.Framework.Sln")]
```
Comment (patriksvensson) [inline tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs:1461]: Expected should be `Statiq.Framework.sln`

**Review by patriksvensson — CHANGES_REQUESTED** [review]

File: tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs (hunk 1 above)
Comment (devlead) [reply tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs:1461]: It does title case for document titles, haven't changed that behavior

Comment (patriksvensson) [reply tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs:1461]: Ah, ok. Didn't know. Thought it was a typo.

**Review by mholo65 — APPROVED** [review]

File: tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs (hunk 1 above)
Comment (daveaglick) [reply tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs:1461]: Thanks for keeping an eye out @patriksvensson!

**Comment by daveaglick** [comment]
Thanks @devlead!

**Pull request merged**